netbox/dcim/models.py

```diff
@@ -2923,9 +2923,9 @@ def save(self, *args, **kwargs):
         # Cache the available_power property on the instance
         kva = self.voltage * self.amperage * (self.max_utilization / 100)
         if self.phase == POWERFEED_PHASE_3PHASE:
-            self.available_power = round(kva * 1.732)
+            self.available_power = round(kva * 1.732) or 1
         else:
-            self.available_power = round(kva)
+            self.available_power = round(kva) or 1
```

Comment:
While this does prevent the error, we're essentially overriding user-input values with an arbitrary static to avoid returning zero. Although I can't imagine a real-world scenario that would result in `available_power` being set to 0 kVA, IMO a more elegant approach would be to address this in the template (`dcim/powerfeed.html`).